### PR TITLE
{phys}[intel/2015b] FDS 6.3.0 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/f/FDS/FDS-6.3.0-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+skipsteps = ['configure', 'install']
+buildininstalldir = True
+
+modextrapaths = {'PATH': 'FDS_Compilation'}
+
+sanity_check_paths = {
+    'files': ['FDS_Compilation/fds'],
+    'dirs': [],
+}
+moduleclass = 'phys'
```
Comment (wpoely86) [inline easybuild/easyconfigs/f/FDS/FDS-6.3.0-intel-2015b.eb]: add newline before

**Comment by wpoely86** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/074a78c93f63bc2e00b6 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/171b9d75d99ec333aec1 for a full test report.

File: easybuild/easyconfigs/f/FDS/FDS-6.3.0-intel-2015b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/f/FDS/FDS-6.3.0-intel-2015b.eb]: done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/203073c289c99a8d6c54 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0c864b6c4749c8992bba for a full test report.

**Comment by boegel** [comment]
going in, thanks for the review @wpoely86!

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e79cb3bbed67e1d27c09 for a full test report.

**Pull request merged**